README.md

```diff
@@ -156,40 +159,41 @@ __Method:__: `autocannon -c 100 -d 40 -p 10 localhost:3000` * 2, taking the seco
 | Koa                | 2.13.0                     | &#10007;     | 54,848        |
 | **Fastify**        | **3.0.0**                  | **&#10003;** | **78,956**    |
 | -                  |                            |              |               |
-| `http.Server`      | 12.18.2	                  | &#10007;     | 70,380        |
+| `http.Server`      | 12.18.2                   | &#10007;     | 70,380        |
 
-Benchmarks taken using https://github.com/fastify/benchmarks. This is a
+Benchmarks taken using <https://github.com/fastify/benchmarks>. This is a
```

Comment:
Why the addition of `<` and `>`?

Reply:
https://github.com/DavidAnson/markdownlint/blob/v0.22.0/doc/Rules.md#md034

We can ad md034 to the ignore list if you prefer? Please advise.

Reply:
Added md034 to ignored rules.

Reply:
We could also do this:

```
Benchmarks taken using [fastify benchmarks](https://github.com/fastify/benchmarks)
```

WDYT?

Reply:
> Rationale: Without angle brackets, the URL isn't converted into a link by many markdown parsers.

Except, they are on GitHub and the tooling we to build our hosted site. So I don't see the utility of the rule.